app/javascript/components/public/locations/Map.jsx

```diff
@@ -1,26 +1,25 @@
-import React from "react";
```

Comment:
the location page map does not seem to work in development

Reply:
Yes, this is an issue I just discovered when having both Vue and React JSX transforms living together. I have to work on a fix.

Reply:
resolving, keeping in the bug task